actors/runtime/src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub mod test_utils;
 macro_rules! wasm_trampoline {
     ($target:ty) => {
         #[no_mangle]
+        #[cfg(target_arch = "wasm32")]
```

Comment:
Oh, disregard my previous comment. The conditional compilation is applied inside the trampoline.